lib/slop/options.rb

```diff
@@ -26,7 +26,7 @@ class Options
     def initialize(**config)
       @options    = []
       @separators = []
-      @banner     = "usage: #{$0} [options]"
+      @banner     = config[:banner] ? config[:banner] : "usage: #{$0} [options]"
```

Comment:
Yep I agree. Do you think you could add that functionality and a test to support `banner: false`?